src/hypertrace/agent/config/__init__.py

```diff
@@ -263,13 +268,17 @@ def __init__(self):  # pylint: disable=R0912,R0915
         self.agent_config.service_name = self.config['service_name']
         self.agent_config.reporting = reporting
         self.agent_config.data_capture = data_capture
-        if self.config['propagation_formats']  == 'TRACECONTEXT':
-            self.agent_config.propagation_formats = config_pb2.PropagationFormat.TRACECONTEXT
-        elif self.config['propagation_formats'] == 'B3':
-            self.agent_config.propagation_formats = config_pb2.PropagationFormat.B3
+        if 'TRACECONTEXT' in self.config['propagation_formats']:
+            self.agent_config.propagation_formats.add(
```

Comment:
This part I could not make it work. This is supposed to be a list but I could not make it work like that and could not decypher the methods for this object. Would you help me with that @rcbjBlueMars ?

Reply:
I've gotten up to this PR for merging all outstanding changes. I'm going to merge in the latest main branch, fix the parsing logic, and  update all tests to correctly use an array of strings.